lcls_tools/common/matlab2py/image/mat_image.py

```diff
@@ -154,21 +160,24 @@ def _unpack_mat_data(self, mat_file):
         self._center_y = data[15][0][0]
         self._filt_stat = data[16][0]
         self._filt_od = data[17][0]
-        self._image_attn = data[18][0][0] 
+        self._image_attn = data[18][0][0]
         self._is_raw = data[19][0][0]
         self._back = data[20][0][0]
 
     def load_mat_image(self, mat_file):
         """Converting super gross .mat image data structure to an object"""
+        if not os.path.isfile(mat_file):
+            raise FileNotFoundError(f"Could not find {mat_file}")
         try:
             self._unpack_mat_data(mat_file)
         except Exception as e:
-            print('error loading mat file: {0}'.format(e))
+            print("error loading mat file {0}: {1}".format(mat_file, e))
```

Comment:
Where did the {1} come from? I'm just not familiar with formatting strings for printing.

Reply:
This looks like a python2 hangover to me. There are 2 arguments that get injected into the print statement and the 0,1 refer to their order in the .format function (0 = mat_file, 1 = exception-object)